src/Routers/office.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
+const useCasesOffice = require('../UseCases/office');
+const auth = require('../Middlewares/auth')
 
-const useCasesOffice = require('../UseCases/office')
+const router = express.Router();
 
-const router = express.router();
 
-router.get('/office', async (request, response) => {
+
+router.get('/office',auth, async (request, response) => {
```

Comment:
@Deveckor los endpoints deben ser en plural.

```suggestion
router.get('/offices',auth, async (request, response) => {
```